fuzz/fuzzers/deframer.rs

```diff
@@ -9,17 +9,18 @@ use rustls::internal::record_layer::RecordLayer;
 use std::io;
 
 fuzz_target!(|data: &[u8]| {
+    let mut buf = <_>::default();
     let mut dfm = deframer::MessageDeframer::default();
     if dfm
-        .read(&mut io::Cursor::new(data))
+        .read(&mut io::Cursor::new(data), &mut buf)
         .is_err()
     {
         return;
     }
-    dfm.has_pending();
+    buf.has_pending();
```

Comment:
This was in the original code too, but I'm not sure why? Doesn't this end up having no effect? Should there be an `assert!` on the result? :thinking:

Reply:
Yes, this is attempting to verify that this function is panic-free, and expose any branches to `panic!` inside to the coverage-driven fuzzer.

Reply:
Gotcha, :+1: